pom.xml

```diff
@@ -64,15 +64,15 @@
     </repositories>
 
     <properties>
-        <cdap.version>4.1.0</cdap.version>
+        <cdap.version>4.3.0</cdap.version>
         <hadoop.version>2.3.0</hadoop.version>
         <junit.version>4.11</junit.version>
         <hydrator.version>1.6.0</hydrator.version>
         <spark.version>1.6.2</spark.version>
         <widgets.dir>widgets</widgets.dir>
         <docs.dir>docs</docs.dir>
         <app.parents>
-            system:cdap-etl-batch[4.0.0,10.0.0-SNAPSHOT),system:cdap-data-pipeline[4.0.0,10.0.0-SNAPSHOT),system:cdap-data-streams[4.0.0,10.0.0-SNAPSHOT)
+            system:cdap-data-pipeline[4.0.0,10.0.0-SNAPSHOT),system:cdap-data-streams[4.0.0,10.0.0-SNAPSHOT)
```

Comment:
Does this only contain the streaming source? If so, cdap-data-pipeline should not be a parent either.

Reply:
fixed